src/cosmos/cosmosRegistry.ts

```diff
@@ -1,16 +1,34 @@
 import { fromBech32 } from '@cosmjs/encoding'
 import { EncodeObject, Registry } from '@cosmjs/proto-signing'
+import { MsgSend } from 'cosmjs-types/cosmos/bank/v1beta1/tx'
 
 import { DepositOpts, TransferOpts, UpgradedRegistry } from './cosmosTypes'
 import { assetFromString } from './cosmosUtils'
 import { MsgDeposit } from './info/proto/thorchainrune/thorchain/v1/x/thorchain/types/msg_deposit'
-import { MsgSend } from './info/proto/thorchainrune/thorchain/v1/x/thorchain/types/msg_send'
+import { MsgSend as thorchainruneMsgSend } from './info/proto/thorchainrune/thorchain/v1/x/thorchain/types/msg_send'
```

Comment:
Since this seems like a class, name should be camel-case `ThorchainRuneMsgSend`

Reply:
done